src/pages/documentation/mobile-core/signal/api-reference.md

```diff
@@ -27,19 +27,18 @@ Flutter
 <Tabs query="platform=flutter&api=extension-version"/> --->
 
 ## registerExtension
+<InlineAlert variant="warning" slots="text"/>
+
+This API is only available in Android and is deprecated in the latest version of the Signal extension. Use [`MobileCore.registerExtensions()`](../api-reference.md#registerextensions) instead.
```

Comment:
`latest` version will change with future releases so let's be specific that it's deprecated with v2.0.0.